index.html

```diff
@@ -1002,7 +1002,7 @@ <h3>Workflows and Exchanges</h3>
 addition to the VC API exchange protocol; for example, an exchange could
 potentially be executable with any of the OID4VCI, OID4VP, DIDComm, and
 VC API exchange protocols. Which protocols are supported by a given workflow
-depend on the workflow service implementation, the complexity of the workflow
+depends on the workflow service implementation, the complexity of the workflow
 the exchange is based on, and the options provided by the coordinator when
```

Comment:
```suggestion
the exchange is based on, and the options provided by the coordinator when
```

Reply:
This got a little weird so I applied a separate fix for it.